doc/source/index.rst

```diff
@@ -117,6 +117,12 @@ The task 'runastrodriz' can be used to reproduce the same Drizzle processing tha
 
    Running Astrodriz <runastrodriz.rst>
 
+HLA API
```

Comment:
The Mission Office has expressed an interest to NOT refer to these new products as HLA, since that implies a product generated by the separate HLA effort.  Instead, we have been asked to refer to this code and the products as 'Enhanced Pipeline Products' in the documentation, at least until some better term is decided upon by the Mission Offices.